Subject: [PATCH] ppc32: Fix sleep on old 101 PowerBook
From: Benjamin Herrenschmidt <benh@kernel.crashing.org>

A typo in the machine table incorrectly mark the 101 PowerBook as
needing explicit callback from the video driver to enable sleep mode. I
did not implement that mecanism for chipsest older than r128, so we need
to mark this machine as always beeing able to sleep for now.

Signed-off-by: Benjamin Herrenschmidt <benh@kernel.crashing.org>

--- a/arch/ppc/platforms/pmac_feature.c	2005-04-30 03:31:35.000000000 +0200
+++ b/arch/ppc/platforms/pmac_feature.c	2005-05-03 13:43:28.000000000 +0200
@@ -2080,7 +2080,7 @@
 	},
 	{	"PowerBook1,1",			"PowerBook 101 (Lombard)",
 		PMAC_TYPE_101_PBOOK,		paddington_features,
-		PMAC_MB_MAY_SLEEP | PMAC_MB_MOBILE
+		PMAC_MB_CAN_SLEEP | PMAC_MB_MOBILE
 	},
 	{	"iMac,1",			"iMac (first generation)",
 		PMAC_TYPE_ORIG_IMAC,		paddington_features,
